### ISSUE TITLE
Flash periodic tables on mobile devices

### ISSUE THREAD

**Issue by mrakitin** [mrakitin/web#23]
On mobile devices Flash periodic tables throw a request to download them. Need to process this case.

**Comment by mrakitin** [comment]
http://www.warmforestflash.com/blog/2010/07/make-your-flash-site-iphone-friendly/

**Comment by mrakitin** [comment]
http://stackoverflow.com/a/449861/4143531:
```html
<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" width="800" height="600" id="flashContent">
  <param name="movie" value="flash.swf" />
  <!--[if !IE]>-->
    <object type="application/x-shockwave-flash" data="flash.swf" width="800" height="600">
  <!--<![endif]-->
      <img src="(...)" alt="Put your alternate content here" />
  <!--[if !IE]>-->
    </object>
  <!--<![endif]-->
</object>
```

**Comment by mrakitin** [comment]
Or http://stackoverflow.com/a/5232972/4143531:
```html
<object type="application/x-shockwave-flash" 
        data="images/flash.swf" 
        width="450" height="960">
  <param name="movie" value="images/flash.swf" />
  <img src="images/header.jpg" />
</object>
```

**Comment by mrakitin** [comment]
Or http://stackoverflow.com/a/5232972/4143531:
```html
<object type="application/x-shockwave-flash" 
        data="images/flash.swf" 
        width="450" height="960">
  <param name="movie" value="images/flash.swf" />
  <img src="images/header.jpg" />
</object>
```

**Comment by mrakitin** [comment]
Fixed via 0b5c1eb.
